lxd-qemu-snap/snapcraft.yaml

```diff
@@ -0,0 +1,213 @@
+name: qemu-for-lxd
+base: core24
+assumes:
+  - snapd2.63
+version: '1'
+grade: devel
+summary: External Qemu for LXD
+description: |-
+ External Qemu for LXD
+
+contact: lxd@lists.canonical.com
+issues: https://github.com/canonical/lxd/issues
+source-code: https://github.com/canonical/lxd
+website: https://ubuntu.com/lxd
+confinement: strict
+
+slots:
+  external-qemu:
+    interface: content
+    content: external-qemu-binaries
+    read:
+      - $SNAP/opt/external-qemu
+
+parts:
+  gpu-2404:
```

Comment:
Is this section needed? What is it used for?

Reply:
It does a cleanup of libraries in snap to prevent collisions with what `gpu-2404` snap provides.